qiita_ware/test/test_processing_pipeline.py

```diff
@@ -271,6 +326,55 @@ def test_get_preprocess_fastq_cmd(self):
         self.assertEqual(obs_cmd_1, exp_cmd_1)
         self.assertEqual(obs_cmd_2, exp_cmd_2)
 
+    def test_get_preprocess_fastq_cmd_per_sample_FASTQ(self):
+        metadata_dict = {
+            'SKB8.640193': {'run_prefix': "sample1", 'primer': 'A',
+                            'barcode': 'A', 'center_name': 'ANL',
+                            'platform': 'ILLUMINA',
+                            'library_construction_protocol': 'A',
+                            'experiment_design_description': 'A'},
+            'SKD8.640184': {'run_prefix': "sample2", 'primer': 'A',
+                            'barcode': 'A', 'center_name': 'ANL',
+                            'platform': 'ILLUMINA',
+                            'library_construction_protocol': 'A',
+                            'experiment_design_description': 'A'}}
+        md_template = pd.DataFrame.from_dict(metadata_dict, orient='index')
+        prep_template = PrepTemplate.create(md_template, Study(1), '16S')
+
+        tmp_dir = mkdtemp()
```

Comment:
I don't think you need this line nor the next two lines, this is already being done in the `setUp` method.